msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -443,17 +445,17 @@ A color to use with and as a background for your outline icons.
 
 The value must be a valid HTML color code starting with '#', for example `#4464ee`.
 
-## copilotExtensions
+## copilotAgents
 
 **Optional** &ndash; Object
 
 Defines one or more agents to Microsoft 365 Copilot. [Declarative agents](/microsoft-365-copilot/extensibility/overview-declarative-agent) are customizations of Microsoft 365 Copilot that run on its same orchestrator and foundation models.
 
 |Name| Type| Maximum size | Required | Description|
 |---|---|---|---|---|
-|`declarativeCopilots`|Array of objects| 1 |✔️| Array of `declarativeCopilot` objects. |
+|`declarativeAgents`|Array of objects| 1 |✔️| Array of `declarativeAgent` objects. |
```

Comment:
|`declarativeAgents`|Array of objects| 1 |✔️| Array of objects defining the declarative agent. |

Reply:
Added